### PR TITLE
give an example of disabling/enabling unattended upgrades

### PR DESCRIPTION
**Pull request by paulgmiller**
Based on https://github.com/denniszielke/container_demos/blob/daa5558a1e731e7577debf93706796cf77538484/operations/set-upgrade-ds.yaml but not using seeperate configmap and sleeping rather than continually looping.

### CODE REVIEW

**Review by palma21 — APPROVED** [review]

**Comment by xhl873** [comment]
nit: disabling in title?

File: examples/unattended-upgrades/daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          command:
+          - /bin/sh
+          - -c
+          - |
+            echo "previous config"
+            less /config
+            # Deactivate upgrades
+            sed -e 's/Update-Package-Lists "1"/Update-Package-Lists "0"/ ; s/Unattended-Upgrade "1"/Unattended-Upgrade "0"/' /config > /config_out
+            # Activate upgrades
+            # sed -e 's/Update-Package-Lists "0"/Update-Package-Lists "1"/ ; s/Unattended-Upgrade "0"/Unattended-Upgrade "1"/' /config > /config_out
```
Comment (charliedmcb) [inline examples/unattended-upgrades/daemonset.yaml:32]: nit: extra commented out line?

File: examples/unattended-upgrades/daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            cat /config_out > /config
+            echo "new config:"
+            less /config
+            sleep 1000000
```
Comment (charliedmcb) [inline examples/unattended-upgrades/daemonset.yaml:36]: Why the sleep 1000000? Will it run again without it/once the sleep is over?

**Review by xhl873 — APPROVED** [review]

**Review by charliedmcb — COMMENTED** [review]
Add a bit more to the PR description? The link you have takes me to the top level of a repo. Is it based on the whole repo, or a specific demo from it?

File: examples/unattended-upgrades/daemonset.yaml (hunk 2 above)
Comment (paulgmiller) [reply examples/unattended-upgrades/daemonset.yaml:36]: so if you don't sleep you run constantly and you lose the stdout logs as it keeps looping. 
Perhaps a better method is to use restartpolicy: onfailure but in these adhoc scripts it's actually hard to catch a failure as bash doesn't carry it thorogh without set -ex

File: examples/unattended-upgrades/daemonset.yaml (hunk 1 above)
Comment (paulgmiller) [reply examples/unattended-upgrades/daemonset.yaml:32]: Nope that's the activate. So if customer wants to turn it backon they edit and flop comments.

**Pull request merged**